tests/foreman/ui/test_reporttemplates.py

```diff
@@ -524,3 +526,97 @@ def test_positive_generate_all_installed_packages_report(
         tree_result = etree.tostring(tree.getroot(), pretty_print=True, method='html').decode()
     assert client.hostname in tree_result
     assert FAKE_1_CUSTOM_PACKAGE in tree_result
+
+
+@pytest.mark.tier2
+@pytest.mark.no_containers
+@pytest.mark.rhel_ver_match('[^6]')
+def test_positive_installable_errata_with_user(
+    session, target_sat, function_org, function_lce, function_location, rhel_contenthost
+):
+    """Generate an Installable Errata report using the Report Template - Available Errata,
+        with the option of 'Installable' with a registered user.
+
+    :id: 036e3ed6-b31b-4481-b591-dae438e4e61d
+
+    :setup: A Host with some applied errata
+
+    :steps:
+        1. Install an outdated package version
+        2. Apply an errata which updates the package
+        3. Downgrade the package impacted by the erratum
+        4. Perform a search for any Available Errata
+        5. Generate an Installable Report from the Available Errata with a registered user
+
+    :expectedresults: A report is generated with the installable errata listed
+
+    :CaseImportance: Medium
+
+    :customerscenario: true
+
+    :BZ: 1726504
+    """
+    activation_key = target_sat.api.ActivationKey(
+        environment=function_lce, organization=function_org
+    ).create()
+    custom_cv = target_sat.api.ContentView(organization=function_org).create()
+    ERRATUM_ID = str(settings.repos.yum_6.errata[2])
+    target_sat.cli_factory.setup_org_for_a_custom_repo(
+        {
+            'url': settings.repos.yum_6.url,
+            'organization-id': function_org.id,
+            'content-view-id': custom_cv.id,
+            'lifecycle-environment-id': function_lce.id,
+            'activationkey-id': activation_key.id,
+        }
+    )
+    result = rhel_contenthost.register(
+        function_org, function_location, activation_key.name, target_sat
+    )
+    assert f'The registered system name is: {rhel_contenthost.hostname}' in result.stdout
+    assert rhel_contenthost.subscribed
+
+    # Remove package if already installed on this host
+    rhel_contenthost.execute(f'yum remove -y {FAKE_1_CUSTOM_PACKAGE_NAME}')
+    # Install the outdated package version
+    rhel_contenthost.execute(r'subscription-manager repos --enable \*')
```

Comment:
Is this needed? I believe the repo should be enabled by the AK (overriden to `enabled` already [here](https://github.com/SatelliteQE/robottelo/blob/db54ac76431e9ba52db3e6c2e3503f735176f9ef/robottelo/host_helpers/cli_factory.py#L687-L689)). 
Please correct me if I am wrong.

Reply:
seems fine to make sure in this case, no harm in the line if it ends up being unnecessary.

Reply:
Agree with Vlad here, Repos already enabled (L564), no need to enable explicitly

Reply:
Removed repo enabled from tr